activerecord/CHANGELOG.md

```diff
@@ -4,6 +4,12 @@
 
     *Federico Martinez*
 
+*   Add `reselect` method. This is a short-hand for `unscope(:select).select(fields)`.
+
+    Fixes #27340.
+
+    *Willian Gustavo Veiga*
+
```

Comment:
Would be great to move this changelog entry to the top of the file to point that this method will be available since the next Rails release.